frontend/src/components/CardDetailsPage.tsx

```diff
@@ -55,6 +59,63 @@ const DetailsCard = ({
   type,
   userSummary,
 }: DetailsCardProps) => {
+  const renderDetailValue = useCallback(
+    (detail: { label: string; value: string | JSX.Element }) => {
+      const { label, value } = detail
+
+      if (
```

Comment:
This works good!
My only concern is with the change of a fallback.

Can we keep it `N/A` if nothing is available to show? 

Current state in staging:
<img width="217" height="178" alt="Screenshot 2025-08-30 at 3 28 17 PM" src="https://github.com/user-attachments/assets/c7cdd116-e13c-462e-8cec-69a352c96974" />